python/src/Wilks_doc.i.in

```diff
@@ -1,66 +1,101 @@
 
-    \Pset (\tilde{q}_{\alpha} N_{\alpha, \beta, i} > q_{\alpha}) > \beta
+    X_{(1)} \leq X_{(2)} \leq \dots \leq X_{(\sampleSize)} 
+
+Given :math:`\alpha`, :math:`\beta` and :math:`i`, the class estimates the minimal size :math:`\sampleSize` such that:
+
+.. math::
 
-where :math:`\tilde{q}_{\alpha} N_{\alpha, \beta, i} = X_{n-i}`."
+    \Prob{X_{(\sampleSize-i)} \geq x_{\alpha}} \geq \beta
+
+Once the minimal size :math:`\sampleSize`  has been estimated, a sample of size :math:`\sampleSize` can be generated from
+:math:`X` and an upper bound of :math:`x_{\alpha}` is estimated by the value of the :math:`X_{(\sampleSize-i)}` on the sample."
 
 // ---------------------------------------------------------------------
 
 %feature("docstring") OT::Wilks::ComputeSampleSize
-"Evaluate the size of the sample.
+"Evaluate the minimum size of the sample.
 
 Parameters
 ----------
 alpha : positive float :math:`< 1`
-    The order of the quantile we want to evaluate.
-beta : positive float :math:`< 1`
-    Confidence on the evaluation of the empirical quantile.
+    Order :math:`\alpha` of the quantile.
+beta : positive float :math:`< 1`,
+    Confidence on the upper bound.
 i : int
-    Rank of the maximum which will evaluate the empirical quantile. Default
-    :math:`i = 0` (maximum of the sample)
+    Index such that :math:`X_{(\sampleSize -i)}` is an upper bound of :math:`x_{\alpha}`
+    with confidence :math:`\beta`.
+    
+    Default value is :math:`i = 0`.
 
 Returns
 -------
-w : int
-    the Wilks number."
+n : int,
+    The minimum size of the sample.
+
+Notes
+-----
+The minimum sample size :math:`\sampleSize` is such that:
+
+.. math::
+
+    \Prob{X_{(\sampleSize-i)} \geq x_{\alpha}} \geq \beta
+"
 
 // ---------------------------------------------------------------------
 
 %feature("docstring") OT::Wilks::computeQuantileBound
-"Evaluate the bound of the quantile.
+"Evaluate an upper bound of a quantile.
 
 Parameters
 ----------
 alpha : positive float :math:`< 1`
-    The order of the quantile we want to evaluate.
+    The order :math:`\alpha` of the quantile.
 beta : positive float :math:`< 1`
-    Confidence on the evaluation of the empirical quantile.
+    Confidence on the upper bound.
 i : int
-    Rank of the maximum which will evaluate the empirical quantile. Default
-    :math:`i = 0` (maximum of the sample)
+    Indexsuch that :math:`X_{(\sampleSize -i)}` is an upper bound of :math:`x_{\alpha}`
+    with confidence :math:`\beta`.
+    
+    Default value is :math:`i = 0`.
 
 Returns
 -------
-q : :class:`~openturns.Point`
-    The estimate of the quantile upper bound for the given quantile level, at
-    the given confidence level and using the given upper statistics."
+upperBound : :class:`~openturns.Point`
+    The estimate of the quantile upper bound.
+
+Notes
+-----
+The method starts by evaluating the minimum size of the sample that ensures that:
+
+.. math::
+
+    \Prob{X_{(\sampleSize-i)} \geq x_{\alpha}} \geq \beta
```

Comment:
Exchanging the order of the expressions is clearer in my opinion.
```suggestion
    \Prob{x_{\alpha} \leq X_{(\sampleSize-i)}} \geq \beta
```

Reply:
ok